BE155

mdziewiur opened the pull request.

Commit:
feat: create EventResponseDto class

create EventResponseDto class and all depedent classes

Commit:
feat: add all dependencies of EventInfoDto class and GeocodingApiDto

add LocationDto, GeocodingApiDto, AttachmentDto classes

Commit:
feat: complete dto clasess related to EventSurvey

complete EventSurveyDto, AnswerRequestDto and AnswerResponseDto classes

Commit:
feat: package dto classes

pacakge dto classes according to entities

Commit:
feat: add Event create and update dtos

add seperate dtos for classes inheriting from Event

Commit:
verify project structure in the scope of dtos

reorder packages, add annotations and any missing dto classes

Norshyee commented on src/main/java/pl/envelo/moovelo/controller/dto/AttachmentDto.java:
```diff
@@ -0,0 +1,11 @@
+
+import lombok.Builder;
+
+@Builder
```
Brakuje @Getter dla wszystkich dto

mdziewiur committed:
fix: adjust Lombok annotation (@Builder and @Getter)

add @Getter to all dto classes, remove @Builder where not necessary and add constructors to these classes
